distage/distage-core/src/test/scala/izumi/distage/dsl/DSLTest.scala

```diff
@@ -67,6 +67,27 @@ class DSLTest extends AnyWordSpec with MkInjector {
       )
     }
 
+    "support annotated parameters after from" in {
```

Comment:
Could you please also add a test that uses the injector to instantiate such a module?

Reply:
Done.